src/components/snapshot/overview/SnapshotOverviewPanel.tsx

```diff
@@ -68,6 +68,7 @@ function SnapshotOverviewPanel(props: SnapshotOverviewPanelProps) {
   const [value, setValue] = React.useState(0);
   const { classes, dispatch, snapshot, userRoles } = props;
   const canEditDescription = userRoles.includes(SnapshotRoles.STEWARD);
+  const canManageUsers = userRoles.includes(SnapshotRoles.STEWARD);
```

Comment:
More of an FYSA, I've got an upcoming change to rename canManageUsers to 'isSteward' because I'm starting to use it in a new component.  If you think it makes sense to introduce that concept in the decisions (especially the ones that need to be made on a more granular basis), run with it in this PR and I'll happily resolve the conflicts in my own branch.  :-)

Reply:
If I were to have a preference, I think I would rather keep the variable called `canManageUsers`, because maybe the conditions for that will change, but the abilities this bool unlocks should not. For example, custodians can manage some users, and not just "isSteward"

Would you be okay with keeping it as is? :)